packages/manager/src/features/VPCs/VPCLanding/VPCLanding.tsx

```diff
@@ -73,6 +75,10 @@ const VPCLanding = () => {
     history.push(VPC_CREATE_ROUTE);
   };
 
+  const isVPCSReadOnly = useRestrictedGlobalGrantCheck({
```

Comment:
Something like `isVPCCreationRestricted` might be a bit more descriptive.

```suggestion
  const isVPCCreationRestricted = useRestrictedGlobalGrantCheck({
```